data/microos/butane/script

```diff
@@ -107,5 +117,15 @@ EOF
 systemctl enable sshd.service
 systemctl enable create_test_file.service
 
+#
+### Networking
+#
+# configure interface for minimal-vm based on sle
+IF=$(find /sys/class/net -type l -not -lname '*virtual*' -printf '%f\n')
```

Comment:
This breaks if there's more than once interface

Reply:
hm.... do we have a test suite with more than 1?

Reply:
No idea.

Reply:
I haven't found any as of now either. So let's just assume there is only 1 interface attached to the VM.

Reply:
:shrug: